packages/lesswrong/server/manualMigrations/2024-05-26-setNotNullClientIds.ts

```diff
@@ -0,0 +1,52 @@
+import { getSqlClientOrThrow } from "../../lib/sql/sqlClient";
+import { registerMigration } from "./migrationUtils";
+
+export const idxName = 'idx_idx_ClientIds_clientId_unique'
+
+registerMigration({
+  name: "setNotNullClientIds",
```

Comment:
I take it you've tested this and verified that it doesn't lock?

Reply:
Yep